packages/experimental/Checkbox/src/Checkbox.types.ts

```diff
@@ -38,6 +39,11 @@ export interface CheckboxProps extends Omit<IViewProps, 'onPress'> {
    */
   checked?: boolean;
 
+  /**
+   * Allows you to set the checkbox to have circular styling.
+   */
+  circular?: boolean;
+
```

Comment:
@chiuam can correct me if I'm wrong, but I think experimental checkbox on macOS is a native module so it can't become circular as of yet. Since that's the case, could we add a `@platform Android, iOS, windows, win32` flag to the end of the comment, or a note that it's not supported on macOS?

Reply:
Done